.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "engines/pisa-0.8.2/pisa"]
+	path = engines/pisa-0.8.2/pisa
+	url = https://github.com/pisa-engine/pisa.git
```

Comment:
can we point to a specific revision here?

I want to make sure build does not get broken too often.